### PR TITLE
Closes #302

### LINKED ISSUE #302: Add Java Logic/Math Operators
# Description

Logic/Math Operators for Java aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/operators.json` to the right Java directory in `web/thesauruses/`
* Edit the operators.json file to accurately reflect the latest major version of Java
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then leave `"code":` blank.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by vinayak0127**
## What GitHub issue does this PR apply to?

Resolves #302 

## Checklist

- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at the documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [x] (If editing Django) I have added or edited any appropriate unit tests for my changes

### CODE REVIEW

File: web/thesauruses/java/operators.json [hunk 1]
```diff
@@ -0,0 +1,191 @@
+{
+  "meta": {
+    "language": "language_id",
```
Comment (geekygirlsarah) [inline web/thesauruses/java/operators.json]: Can you fill out these 3 lines to reflect that it's Java, and the latest version of Java?

File: web/thesauruses/java/operators.json [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      "code": "%="
+    },
+    "increment": {
+      "code": "++"
+    },
+    "decrement": {
+      "code": "--"
+    },
+    "exponential": {
+      "code": "",
```
Comment (geekygirlsarah) [inline web/thesauruses/java/operators.json]: For the `not-implemented` concepts, can you remove the `code` lines? It'll help clean up the file as they're not needed here.

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
I think if you make these tweaks, I'll merge it in!

**Comment by vinayak0127** [comment]
Hey! @geekygirlsarah I have updated the changes can you review them.

**Review by geekygirlsarah — APPROVED** [review]

**Comment by geekygirlsarah** [comment]
With the new changes, I think this works. I'll merge it in!

Don't forget to sign up for Hacktoberfest if you want. This counts towards the shirt/swag!

**Pull request merged**